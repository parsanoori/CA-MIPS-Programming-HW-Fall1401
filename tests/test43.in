88
386069911
716316828
394954572
933756374
767862909
464441477
48542060
952175400
953588868
693869712
847101611
513726268
921014134
269109
996544556
946506791
816175997
941793117
337272568
289231360
164146174
975768961
631855326
814699210
371135811
556958016
320460319
701107381
626808821
402904792
62249090
472449630
945823332
278584103
655629978
43101213
151134164
630967200
612401160
951544920
453690327
327829523
238041234
367941848
588793053
843343008
394105586
178526104
835508712
699347791
395491160
843620291
332853613
380233435
602691437
1044642347
453170964
743926130
945980624
356362048
977149909
592587812
633311166
270610805
396104296
417724962
955437532
951686779
425508873
119473514
463899788
543128080
1032958252
655439451
693406321
765116001
127139534
364090525
627385493
505778397
599530905
561965085
551895871
108659200
593336634
156045143
866800061
187320323