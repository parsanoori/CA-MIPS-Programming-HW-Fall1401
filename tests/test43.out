269109
43101213
48542060
62249090
108659200
119473514
127139534
151134164
156045143
164146174
178526104
187320323
238041234
270610805
278584103
289231360
320460319
327829523
332853613
337272568
356362048
364090525
367941848
371135811
380233435
386069911
394105586
394954572
395491160
396104296
402904792
417724962
425508873
453170964
453690327
463899788
464441477
472449630
505778397
513726268
543128080
551895871
556958016
561965085
588793053
592587812
593336634
599530905
602691437
612401160
626808821
627385493
630967200
631855326
633311166
655439451
655629978
693406321
693869712
699347791
701107381
716316828
743926130
765116001
767862909
814699210
816175997
835508712
843343008
843620291
847101611
866800061
921014134
933756374
941793117
945823332
945980624
946506791
951544920
951686779
952175400
953588868
955437532
975768961
977149909
996544556
1032958252
1044642347